gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
```

Comment:
Can you use the `all` distribution please?

Reply:
Actually, Gradle sources could be attached to projects by IDEA for now. I can still use `all` if you want.

Reply:
Good to know. Leave it using `bin` for now and if something doesn't work we can always restore `all` without having to release a new version